src/components/QuranReader/QuranReaderView.tsx

```diff
@@ -20,16 +19,7 @@ interface Props {
   quranReaderStyles: QuranReaderStyles;
 }
 
-const QuranReaderView: React.FC<Props> = ({
-  isTafsirData,
-  isSelectedTafsirData,
-  isReadingPreference,
-  verses,
-  quranReaderStyles,
-}) => {
-  if (isTafsirData || isSelectedTafsirData) {
-    return <TafsirView verse={verses[0]} />;
-  }
```

Comment:
need to do follow up PR to remove TafsirView from QuranReader, there's still props related to tafsir, such as `isSelectedTafsirData` and `isTafsirData`

Reply:
👍